exercises/sum.cpp

```diff
@@ -6,3 +6,20 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+using namespace std;
+
+int main() {
+    int n, sum = 0;
+
+    cout << "Enter a positive integer: ";
+    cin >> n;
+
+    for (int i = 1; i <= n; ++i) {
+        sum += i;
+    }
```

Comment:
non era questo l'intento dell'esercizio, bisognava scrivere un programma che prendesse due valori in input ed eseguisse la somma

comunque l'importante è l'aver usato git e github per svolgerlo :+1: